app/models/announcement_notifier.rb

```diff
@@ -2,6 +2,11 @@
 
 class AnnouncementNotifier
   def call(announce)
+    return if announce.wip? || announce.published_at.present?
```

Comment:
すみません、自分でも忘れていたんですが、こう書けたと思います。

```suggestion
    return if announce.wip? || announce.published_at?
```

Reply:
@komagata 
こちら修正しました！